src/GafferImageUI/ImageGadget.cpp

```diff
@@ -75,6 +78,401 @@ using namespace GafferUI;
 using namespace GafferImage;
 using namespace GafferImageUI;
 
+namespace
+{
+
+// These supporting functions, along with RenderTexture itself, contain code copied from ViewportGadget and
+// rearranged. It's expressed in such a way that we could use it as a drop-in replacement for the code in
+// ViewportGadget if we wanted to avoid the duplication - but this would require exposing it more than we
+// are currently comfortable with.
+void renderUnitSquare( const IECoreGL::Shader::Parameter *pParameter, const IECoreGL::Shader::Parameter *uvParameter = nullptr )
+{
+	static float rectPBufferData[12] = { -1, -1, 0,  -1, 1, 0,  1, -1, 0,  1, 1, 0 };
+	static float rectUvBufferData[12] = { 0, 0,  0, 1,  1, 0,  1, 1 };
+	static IECoreGL::BufferPtr g_rectPBuffer = new IECoreGL::Buffer( rectPBufferData, sizeof( float ) * 12 );
```

Comment:
I'm a bit wary of this because it seems like we'll be destroying the GL resource during the destruction of statics at program exit, when there won't be a GL context around. I see we have the same in ViewportGadget though, so maybe it's OK somehow?

Reply:
Seems to be working fine ... do you have an alternate way you'd prefer me to do it?